resources/qml/RoomList.qml

```diff
@@ -184,6 +184,11 @@ Page {
                 onTriggered: newTag.show()
             }
 
+            Platform.MenuItem {
```

Comment:
This should probably be above the tags menu (don't forget to update line 157 then). Alternatively you could put it inside the 3 dot menu in the rooms top bar.